redbot/cogs/cleanup/cleanup.py

```diff
@@ -10,12 +10,18 @@
 from redbot.core.utils.chat_formatting import humanize_number
 from redbot.core.utils.mod import slow_deletion, mass_purge
 from redbot.core.utils.predicates import MessagePredicate
+from .checks import check_self_permissions
 from .converters import PositiveInt, RawMessageIds, positive_int
 
 _ = Translator("Cleanup", __file__)
 
 log = logging.getLogger("red.cleanup")
 
+LINKS = re.compile(r"http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*(),]|(?:%[0-9a-fA-F][0-9a-fA-F]))+")
```

Comment:
We have a regex for links in `redbot.core.utils.common_filters.URL_RE` which could be used here.